v0x04/utils.py

```diff
@@ -65,6 +65,25 @@ def request_port_stats(controller, switch):
     return multipart_request.header.xid
 
 
+def update_table_list(controller, switch):
```

Comment:
This name doesn't reflect what it does, can you rename it as `request_table_stats`?